### DISCUSSION TITLE
when GPT 4.0?

### DISCUSSION THREAD

**Discussion by kingofkillers91** [n3d1117/chatgpt-telegram-bot#68] [Q&A] [answered]
Will you implement it?

**Comment by n3d1117** [comment, marked as answer]
Yes! As soon as its APIs are availble :)

**Reply by kingofkillers91** [reply]
Any ETA for the API  to be released that you know of?

**Reply by n3d1117** [reply]
> Any ETA for the API to be released that you know of?

Not that I know of. There's a waitlist at https://openai.com/waitlist/gpt-4-api

**Reply by kingofkillers91** [reply]
I see. Thanks!!
